doc/users/install.rst

```diff
@@ -76,8 +76,14 @@ nose_ installed, then do the following::
 
 you can also test with nosetests::
 
-    nosetests --with-doctest /software/nipy-repo/masternipype/nipype
-    --exclude=external --exclude=testing
+    nosetests --with-doctest <installation filepath>/nipype  --exclude=external --exclude=testing
```

Comment:
if nipype is already installed this can now be done with: `nosetests --with-doctest nipype`